components/file_limit_sticky_banner/index.tsx

```diff
@@ -42,7 +42,7 @@ border: 1px solid #FFBC1F;
 border-radius: 4px;
 background: linear-gradient(0deg, rgba(255, 212, 112, 0.16), rgba(255, 212, 112, 0.16)), #FFFFFF;
 padding: 16px 17px;
-color: var(--center-channel-color);
+color: var(--center-channel-text);
```

Comment:
@pvev Although we have closed this pr, as you were using `-center-channel-color` earlier, it is the right color, we don’t know `--center-channel-text` got introduced but I think we need to get rid of this.

Reply:
ok, but we need to keep in mind that one is used in several places.